nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutUDP.java

```diff
@@ -26,152 +26,107 @@
     private final static int VALID_LARGE_FILE_SIZE = 32768;
-    private final static int VALID_SMALL_FILE_SIZE = 64;
     private final static int INVALID_LARGE_FILE_SIZE = 1_000_000;
-    private final static int LOAD_TEST_ITERATIONS = 500;
-    private final static int LOAD_TEST_THREAD_COUNT = 1;
-    private final static int DEFAULT_ITERATIONS = 1;
-    private final static int DEFAULT_THREAD_COUNT = 1;
     private final static char CONTENT_CHAR = 'x';
     private final static int DATA_WAIT_PERIOD = 50;
-    private final static int DEFAULT_TEST_TIMEOUT_PERIOD = 10000;
-    private final static int LONG_TEST_TIMEOUT_PERIOD = 30000;
+    private final static String[] EMPTY_FILE = { "" };
+    private final static String[] VALID_FILES = { "FIRST", "SECOND", "12345678", "343424222", "!@£$%^&*()_+:|{}[];\\" };
 
     private TestRunner runner;
     private int port;
     private EventServer eventServer;
     private BlockingQueue<ByteArrayMessage> messages;
 
-
-    // Test Data
-    private final static String[] EMPTY_FILE = { "" };
-    private final static String[] VALID_FILES = { "abcdefghijklmnopqrstuvwxyz", "zyxwvutsrqponmlkjihgfedcba", "12345678", "343424222", "!@£$%^&*()_+:|{}[];\\" };
-
-    @Before
+    @BeforeEach
     public void setup() throws Exception {
         runner = TestRunners.newTestRunner(PutUDP.class);
         runner.setVariable(SERVER_VARIABLE, UDP_SERVER_ADDRESS);
         port = NetworkUtils.getAvailableUdpPort();
         createTestServer(port, VALID_LARGE_FILE_SIZE);
     }
 
-    private void createTestServer(final int port, final int frameSize) throws Exception {
-        messages = new LinkedBlockingQueue<>();
-        final byte[] delimiter = DELIMITER.getBytes(CHARSET);
-        final InetAddress listenAddress = InetAddress.getByName(UDP_SERVER_ADDRESS);
-        NettyEventServerFactory serverFactory = new ByteArrayMessageNettyEventServerFactory(
-                runner.getLogger(), listenAddress, port, TransportProtocol.UDP, delimiter, frameSize, messages);
-        serverFactory.setSocketReceiveBuffer(MAX_FRAME_LENGTH);
-        serverFactory.setShutdownQuietPeriod(ShutdownQuietPeriod.QUICK.getDuration());
-        serverFactory.setShutdownTimeout(ShutdownTimeout.QUICK.getDuration());
-        eventServer = serverFactory.getEventServer();
-    }
-
-    @After
+    @AfterEach
     public void cleanup() {
         runner.shutdown();
         removeTestServer();
     }
 
-    private void removeTestServer() {
-        if (eventServer != null) {
-            eventServer.shutdown();
-            eventServer = null;
-        }
-    }
-
-    @Test(timeout = DEFAULT_TEST_TIMEOUT_PERIOD)
-    public void testValidFiles() throws Exception {
-        configureProperties(UDP_SERVER_ADDRESS);
-        sendTestData(VALID_FILES);
-        checkReceivedAllData(VALID_FILES);
-        checkInputQueueIsEmpty();
-    }
-
-    @Test(timeout = DEFAULT_TEST_TIMEOUT_PERIOD)
-    public void testValidFilesEL() throws Exception {
```

Comment:
Trying to decide if this EL case is useful.

Reply:
I considered that, but running through the entire process seemed less than useful just to verify support for expression language. Other properties also support expression language, and are not checked, so it does not seem to add much value in this case. It is also a property defined on `AbstractPutEventProcessor`, so that would be a better place to implement expression language checks.